open-metadata-resources/open-metadata-deployment/charts/vdc/values.yaml

```diff
@@ -91,74 +71,95 @@ image:
     pullPolicy: Always
 
   egeria:
-    repository: planetf1/egeria-egeriavdc
+    repository: localhost:5000/odpi/egeria
     tag: 1.1-SNAPSHOT
     pullPolicy: Always
 
   gaian:
-    repository: planetf1/gaian-egeriavdc
+    repository: localhost:5000/odpi/gaian-egeriavdc
     tag: 1.1-SNAPSHOT
     pullPolicy: Always
 
   atlas:
-    repository: planetf1/atlas-1773-egeria
-    tag: 1.1-SNAPSHOT
-    pullPolicy: Always
-
-  atlasinit:
-    repository: planetf1/atlas-init-egeria
+    repository: localhost:5000/odpi/apache-atlas
     tag: 1.1-SNAPSHOT
     pullPolicy: Always
 
   kafkamonitor:
-    repository: planetf1/kafka-monitor-egeriavdc
-    tag: 1.1-SNAPSHOT
-    pullPolicy: Always
-
-  egeriainit:
-    repository: planetf1/egeria-init-egeria
-    tag: 1.1-SNAPSHOT
-    pullPolicy: Always
-
-  igcproxyinit:
-    repository: planetf1/igcproxy-init-egeriavdc
-    tag: 1.1-SNAPSHOT
-    pullPolicy: Always
-
-  postgresqlinit:
-    repository: planetf1/postgresql-init-egeria
+    repository: localhost:5000/odpi/kafka-monitor-egeriavdc
     tag: 1.1-SNAPSHOT
     pullPolicy: Always
 
   postgresql:
-    repository: planetf1/postgresql-egeriavdc
-    tag: 1.1-SNAPSHOT
-    pullPolicy: Always
+    repository: postgres
+    tag: alpine
+    pullPolicy: IfNotPresent
 
   omrsmonitor:
-    repository: planetf1/omrsmonitor-egeriavdc
+    repository: localhost:5000/odpi/omrsmonitor-egeriavdc
     tag: 1.1-SNAPSHOT
     pullPolicy: Always
 
   ldapinit:
-    repository: planetf1/ldap-init-egeriavdc
+    repository: localhost:5000/odpi/ldap-init-egeriavdc
     tag: 1.1-SNAPSHOT
     pullPolicy: Always
 
+  # NOTE: this image is only used if the ibmigc.external is set to false; otherwise it is ignored and
+  # the k8s-external, pre-existing environment defined under ibmigc... is used
+  ibmigc:
+    repository: localhost:5000/ibm/infosvr
+    tag: v11.7.0.2
+    pullPolicy: IfNotPresent
+
+  configure:
+    repository: localhost:5000/odpi/egeria-configure
+    tag: 1.1-SNAPSHOT
+    pullPolicy: IfNotPresent
+
   nameOverride: ""
   fullnameOverride: ""
 
-postgres:
-  password: passw0rd
-
-# Egeria depends on kafka, but it is not exposed externally
-# The sample also requires auto topic creation
+# Whether to use a k8s cluster-internal Kafka or an external one -- if using internal,
+# see 'cp-kafka' below for config options
 kafka:
+  internal:
+    enabled: true
   external:
-    enabled: false
-    type: NodePort
+    hostname: ""
+    ip: ""
+    port: ""
```

Comment:
will need credentials I think (mentioned above). i can figure this out in a future PR as needed